ShopperInsights/src/test/java/com/braintreepayments/api/ShopperInsightsClientUnitTest.kt

```diff
@@ -17,36 +20,75 @@ import org.junit.Test
  */
 class ShopperInsightsClientUnitTest {
 
+    private val context: Context = mockk(relaxed = true)
+    private val applicationContext: Context = mockk(relaxed = true)
     private lateinit var sut: ShopperInsightsClient
     private lateinit var paymentApi: PaymentReadyApi
     private lateinit var braintreeClient: BraintreeClient
+    private lateinit var deviceInspector: DeviceInspector
 
     @Before
     fun beforeEach() {
         paymentApi = mockk(relaxed = true)
         braintreeClient = mockk(relaxed = true)
-        sut = ShopperInsightsClient(paymentApi, braintreeClient)
+        deviceInspector = mockk(relaxed = true)
+        every { context.applicationContext } returns applicationContext
+        sut = ShopperInsightsClient(paymentApi, braintreeClient, deviceInspector)
     }
 
     /**
      * Tests if the getRecommendedPaymentMethods method returns paypal and venmo recommendations
      * when providing a shopping insight request.
      */
     @Test
-    fun testGetRecommendedPaymentMethods_returnsDefaultRecommendations() {
+    fun testGetRecommendedPaymentMethods_noInstalledApps_returnsDefaultRecommendations() {
+        every { deviceInspector.isVenmoInstalled(applicationContext) } returns false
+        every { deviceInspector.isPayPalInstalled(applicationContext) } returns false
+
+        val request = ShopperInsightsRequest("fake-email", null)
+        sut.getRecommendedPaymentMethods(context, request) { result ->
+            assertNotNull(result)
+            val successResult = assertIs<ShopperInsightsResult.Success>(result)
+            assertNotNull(successResult.response.isPayPalRecommended)
```

Comment:
Let's assert false here for now, since that is what the API is hard coded to do - and then we won't forget to update these tests based on the actual results in the future.